### DISCUSSION TITLE
How to use the connection migration

### DISCUSSION THREAD

**Discussion by LLL2000L** [microsoft/msquic#3956] [Q&A] [answered]
How can I use the connection migration feature or see the effect of QUIC connection migration? I would appreciate your guidance. Thank you.

**Comment by nibanks** [comment, marked as answer]
We support migration on the server side, and you don't need to do anything to use it. We don't yet support it on the client side yet.

**Comment by madhanraj** [comment]
Hello @nibanks , do you mean client-side migration is enabled by default, correct? I.e. change in client IP address. 

If the server interfaces are in the same device (network box or UE), do you see any technical difficulties in the server-side migration?

**Reply by nibanks** [reply]
No, we have no client-side migration today.

**Comment by arunb2001** [comment]
Hi @nibanks , I have a Linux machine as client which has both ethernet and Wi-Fi interface. When it connects to server (also Linux) and starts sending data the Wi-Fi is disconnected and uses Ethernet connection. However, the client stops sending data and shuts down automatically. I would like to ask that is this the expected behaviour for MSQUIC ?

I am using the sample MSQUIC client and server code for this.

**Reply by nibanks** [reply]
Yes, since we don't have client migration support yet.
